scripts/alert_emails/send-test-email.ts

```diff
@@ -29,7 +29,11 @@ async function main(emailAddress: string) {
     await cm.sendClassicEmail(data);
     console.info(`Email sent to ${emailAddress}.`);
   } catch (err) {
-    console.error(err);
+    if (isInvalidEmailError(err)) {
+      console.log(`Invalid email: "${emailAddress}"`);
+    } else {
+      console.error(`Error sending email to "${emailAddress}"`, err);
```

Comment:
Does this print something useful, or just like "[Object object]" ?

I'm worried that by using a class, we may lose visibility into the actual error details.  We may need to implement toString() or something.

 (I don't actually care about this test code, but I care about the similar error handling code in send_emails.ts)

Although it is verbose, it's nice that the existing log message included both our request and the error details.  Makes it straightforward to diagnose issues.

Reply:
This is the output for a different error (I passed a bad token to check):

```
$ yarn send-test-email pnavarrc@gmail.com      
yarn run v1.22.4
$ ts-node-script -r tsconfig-paths/register scripts/alert_emails/send-test-email.ts pnavarrc@gmail.com
Error sending email to "pnavarrc@gmail.com" CampaignMonitorException {
  Code: 50,
  Message: 'Must supply a valid HTTP Basic Authorization header' }
error Command failed with exit code 1.
```

I can return the original error (which includes the response) instead of the parsed response for unknown errors.

Reply:
Updated: It shows the entire error now, I tested with the test script after updating